DOC.md

```diff
@@ -221,8 +221,8 @@ s("trigger", {
 })
 ```
 The above snippet will behave as follows:
-- After expansion, we will be at InsertNode 1.
 - After jumping forward, we will be at InsertNode 2.
+- After expansion, we will be at InsertNode 1.
```

Comment:
I don't think this should be moved down, we want to demonstrate a sequence of actions (expand -> jump -> jump), these shouldn't follow the order of the insertNodes in the snippet.

Would you make the list numbered to express that more clearly?

Reply:
I changed the order because, after expansion the snippets gives
```
After jumping forward, we will be at ->
After expansion, we will be at ->
After jumping forward again, we will be at ->
```
Should we follow this order?

Reply:
No, I don't think so, the list should show where the cursor is after a given action, the first action is `expand`, followed by two jumps.

It makes more sense to have the list ordered by the actions than by the position of the lines in the snippet I think.